### DISCUSSION TITLE
Fast package rebuilding inside cypress

### DISCUSSION THREAD

**Discussion by senpl** [cypress-io/cypress#29193] [Questions and Help] [answered]
I started fixing some issue in Cypress sourcecode. But to check my changes I had to do:
`yarn postinstall` which takes like 50 seconds.

Is there are faster way to rebuild cypress with only one package changed???
I know I could rebuild package inside 'packages\driver' but how to make that changes into whole project fast.

**Comment by portyu9** [comment]
Hi @senpl 

You do not need to run `yarn postinstall` after every change.

For changes under `packages/driver`, the current Cypress repo workflow is to run the watcher from the repository root:

```bash
yarn watch
```

Or, if you only need the bundle that consumes the driver, use the narrower watcher:

```bash
yarn workspace @packages/runner watch
```

`packages/driver` is bundled through `@packages/runner`, so the watcher will pick up your driver changes and rebuild the bundle automatically.

To work specifically on the driver's Cypress tests, you can then use:

```bash
yarn workspace @packages/driver cypress:open
```

`yarn postinstall` is useful for the initial repository/dependency setup, but it should not be necessary for each edit/retest cycle.

Please mark this answer as accepted if it helped, thank you.

**Comment by senpl** [comment, marked as answer]
https://stackoverflow.com/questions/78223372/fast-package-rebuilding-inside-cypress
This what worked for me around 2 years ago.
